scripts/configure.js

```diff
@@ -62,43 +66,54 @@ if (osName === 'Darwin') {
     process.exit(1);
   }
 
-  const installedHeadersDir = getFreeBSDHeadersDir(lldbVersion);
-  if (installedHeadersDir === undefined) {
+  console.log(`Installing llnode for ${lldbExe}, lldb version ${lldbVersion}`);
+  const installedDir = getFreeBSDInstallDir(lldbVersion);
+  if (installedDir === undefined) {
     // As this is a BSD we know this system is in an improper state
     // So we can exit with an error
     console.log('The system isn\'t set up correcly.');
     console.log('Try `pkg install llvm39');
     console.log('And `ln -s /usr/local/bin/lldb39 /usr/bin/lldb`');
     process.exit(1);
   } else {
-    lldbIncludeDir = installedHeadersDir;
+    lldbInstallDir = installedDir;
   }
-
 }
 
-console.log(`Installing llnode for ${lldbExe}, lldb version ${lldbVersion}`);
+// This should be equivalent to `llvm-config --includedir`/lldb
+function getLldbHeadersPath(lldbInstallDir) {
+  return path.join(lldbInstallDir, 'include', 'lldb');
+}
 
-// Check out source code of the LLDB that is compatible with OS X's default lldb
+// Check out source code of the LLDB for headers
 // TODO: The llvm project is probably moving to github soon at that point we
 // should stop using the mirror.
-if (lldbHeadersBranch != undefined) {
-  console.log('Cloning lldb from ' + lldbHeadersBranch);
-  child_process.execSync(`rm -rf ${lldbIncludeDir}`);
-  child_process.execFileSync('git',
-    ['clone', '--depth=1', '-b', lldbHeadersBranch,
-      'https://github.com/llvm-mirror/lldb.git', lldbIncludeDir],
-    {cwd: buildDir});
+function cloneHeaders(lldbHeadersBranch, lldbInstallDir, buildDir) {
+  const lldbHeaders = getLldbHeadersPath(lldbInstallDir);
+  if (!fs.existsSync(lldbHeaders)) {
+    child_process.execSync(`rm -rf ${lldbInstallDir}`);
```

Comment:
Looking at this again, won't this fail horribly if `lldbInstallDir` contains spaces?

Reply:
@bnoordhuis Ouch..I think maybe we can just skip deleting the directory altogether. In case there are any stale links, the user can totally remove them by hand..